packages/atomic/src/components/search/search-box-suggestions/suggestions-common.ts

```diff
@@ -3,41 +3,136 @@ import {VNode} from '@stencil/core';
 import {buildCustomEvent} from '../../../utils/event-utils';
 import {closest} from '../../../utils/utils';
 import {Bindings} from '../atomic-search-interface/atomic-search-interface';
+
+/**
+ * Element which will be rendered in the list of suggestions.
+ */
 export interface SearchBoxSuggestionElement {
+  /**
+   * Stable identity which enables Stencil to reuse DOM elements for better performance.
+   * The best way to pick a key is to use a string that uniquely identifies that list item among its siblings (often your data will already have IDs).
+   */
   key: string;
+  /**
+   * Rendered content of the element.
+   */
   content: Element | VNode;
+  /**
+   * Hook called when the selection is selected.
+   * @param e DOM event.
```

Comment:
I think we should detail that this is where somebody should execute other actions such as selecting the suggestion. Maybe part of the query suggestion component could be used as an example.

Reply:
The example code I will be using in the usage article utilizes this function. Do you think that should suffice?

Reply:
Perhaps the reference doc could have a link to the usage article.